.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,55 @@
+name: Nightly
+on:
+  schedule:
+    - cron: '0 1 * * *' # Nightly at 1am
+
+defaults:
+  run:
+    shell: bash
+jobs:
+  nightly:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ macOS-latest, ubuntu-latest, windows-latest ]
```

Comment:
Shall we avoid "-latest"? In my experience, it suddenly stops working when GitHub infrastructure changes what is latest.

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners

Reply:
Done